### PR TITLE
CRM-19380 allow for multiple from email addresses but only one per do…

### PR DESCRIPTION
**Pull request by seamuslee001**
…main

### CODE REVIEW

**Comment by seamuslee001** [comment]
@eileenmcnaughton thoughts Eileen?

File: CRM/Core/OptionValue.php [hunk 1]
```diff
@@ -258,15 +258,22 @@ public static function addOptionValue(&$params, &$groupParams, &$action, &$optio
    * @param int $optionGroupID
    * @param string $fieldName
    *   The name of the field in the DAO.
+   * @param bool $domainSpecific
+   *   Dos this option groujp allow for duplicate options that are
```
Comment (eileenmcnaughton) [inline CRM/Core/OptionValue.php]: !! Dos this read right to you?:-)

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by seamuslee001** [comment]
@eileenmcnaughton should this get merged?

**Comment by eileenmcnaughton** [comment]
This seems to make sense - just adding the admin side of storing domain specific settings. Needs someone to give it a quick test though

**Comment by monishdeb** [comment]
Tested. Merging

**Pull request merged**